src/components/ui/rewardsCalculatorDialog.js

```diff
@@ -106,23 +117,55 @@ const RewardsCalculatorDialog = ({
             min="0"
             max="4"
             ref={yearsRef}
-            onChange={handleYearsInput}
+            onChange={event => handleYearsInput(Number(event.target.value))}
             placeholder={`${t("years")} (4 years max)`}
           />
-          {inputIQ && years && aprDividedByLockPeriod ? (
-            <StyledDivContainer className="shadow-sm">
-              <p className="mb-0">
-                {" "}
-                {t("you_will_get")}:{" "}
-                <strong>{Number(expectedIQ).toFixed(2)} IQ</strong>
-              </p>
-              <p className="mb-0">
-                {" "}
-                {t("expected_apr")}:{" "}
-                <strong>{Number(aprDividedByLockPeriod).toFixed(2)} %</strong>
-              </p>
-            </StyledDivContainer>
-          ) : null}
+          <div className="d-flex flex-row justify-content-center p-2">
+            <StyledButton
+              onClick={() => handleYearsInput(0.0833334)}
+              size="sm"
+              variant="light"
+              className="m-1 shadow-sm"
+            >
+              1 month
```

Comment:
I would avoid 1 - 3 - 6 months buttons. We want holders to hold the longest possible so they can get the best APR.

Buttons could be 1 - 2 - 4 years.